src/renderers/shared/fiber/ReactChildFiber.js

```diff
@@ -163,6 +168,10 @@ function createFirstChild(returnFiber, existingChild, newChildren, priority) {
 
 // TODO: This API won't work because we'll need to transfer the side-effects of
 // unmounting children to the returnFiber.
-exports.reconcileChildFibers = function(returnFiber : Fiber, currentFirstChild : ?Fiber, newChildren : ReactNodeList, priority : PriorityLevel) : ?Fiber {
+exports.reconcileChildFibers = function(
+  returnFiber : Fiber, 
+  currentFirstChild : ?Fiber, 
+  newChildren : ReactNodeList, 
+  priority : PriorityLevel) : ?Fiber {
```

Comment:
You can just add a commit to your fork branch and it will show up here. GitHub supports "squash and merge" right now so there's no need to squash commits from your end.